components/home/Testimonials.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <section>
+    <v-container>
+      <v-carousel
+        :current-slide="currentSlide"
+        :transition-name="transitionName"
+        class="carousel-container"
+      >
+        <div
+          v-for="(slide, index) in slides"
+          :key="'testimonial-carousel-' + index"
+        >
+          <v-stack md-direction="column" align="center" md-align="start">
+            <v-image src="/images/moon.svg" class="testimonial-illustration" />
+            <v-stack direction="column">
+              <v-heading :level="4" class="testimonial-quote">
+                “{{ slide.quote }}”
+              </v-heading>
+              <v-stack
+                md-direction="column"
+                style="margin-top: 1.5rem"
+                align="baseline"
+              >
+                <v-text class="testimonial-quoted-by">
+                  {{ slide.quotedBy }},
+                </v-text>
+                <v-text color="secondary" class="designation">
+                  {{ slide.designation }}
+                </v-text>
+              </v-stack>
+              <v-image :src="slide.companyLogo" class="company-logo" />
+            </v-stack>
+          </v-stack>
+        </div>
+      </v-carousel>
+    </v-container>
+  </section>
+</template>
+
+<script>
+import VCarousel from '../lib/VCarousel.vue'
```

Comment:
- Use root-relative imports for internal modules.
- Add a line break after all imports.
- Sort `components` in the component options alphabetically.

Reply:
@punaifang Done all the required changes.